### DISCUSSION TITLE
R Consortium grant application?

### DISCUSSION THREAD

**Discussion by smach** [extendr/extendr#432] [Ideas]
Hi, just wondering if anyone on this team has thought about applying for an R Consortium grant? Sorry if this is obvious and you already have, but seemed worth mentioning! Applications are now open for the next round of funding, and this seems like a project that will “strengthen and improve the infrastructure supporting the R Ecosystem.”
https://www.r-consortium.org/all-projects/call-for-proposals

**Comment by andy-thomason** [comment]
Hi, thanks for asking.

I have already taken advantage of this to do some of the early necessary work, but others on the team might benefit from
doing some specific projects. I found this very useful and we should probably put a "sponsored by R Consortium" message
in the README.

Examples might be:
  Support for blockchain data sources - many of these are Rust-based.
  A package for the SIMD maths library (I have a very fast one in a PR in the core rust SIMD repo but it needs some love).
  Internet of things data capturing - for example monitoring temperature on a Raspberry PI.
  3D plotting using OpenGL or WebGPU.

If any of these take your fancy, I can help you get started, but don't have a lot of time at the moment myself.
